### PR TITLE
Adjust the design of determining the fields and other aspects

### PR DESCRIPTION
**Pull request by younies**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
 # Overview
 
 * Time Duration is how long something lasts, from the start to end. It can be represented by a single time unit or multiple ones. 
-  - For example,
+  + For example,
```
Comment (ryzokuken) [inline README.md:16]: Did you change the bullets on purpose? `+` is not a bullet point in markdown, I would suggest that you either revert back to `-` or use `*`.

File: README.md [hunk 2]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
 
 * Time Duration is how long something lasts, from the start to end. It can be represented by a single time unit or multiple ones. 
-  - For example,
+  + For example, 
     - 10000 seconds
     - 2 hours 46 minutes 40 seconds
 
 * Every locale has its own way to format duration. 
-  - For example:
+  + For example:
```
Comment (ryzokuken) [inline README.md:21]: Same as above.

File: README.md [hunk 3]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
     - en-US: 1 hour, 46 minutes and 40 seconds
     - fr-FR: 1 heure, 46 minutes et 40 secondes
 
 * There are multiple widths for the time duration.
-  - For example, wide and short
+  + For example, wide and short
```
Comment (ryzokuken) [inline README.md:26]: Same as above.

File: README.md [hunk 4]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
     - 1 hour, 46 minutes and 40 seconds → Wide
     - 1 hr, 46 min, 40 sec → Short
 
 # Motivation
 
 * Users need all types of duration format depending on the requirements of their application. For example, to show how long a flight takes, the duration should be in Short or Narrow format
-  - 1 hr 40 min 60 sec → Short 
-  - 1 h 40 m 60 sec  → Narrow
+  + 1 hr 40 min 60 sec → Short 
+  + 1 h 40 m 60 sec  → Narrow
```
Comment (ryzokuken) [inline README.md:34]: Same as above.

File: README.md [hunk 5]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
 
 # Requirements & Design
 
 In this section, we are going to illustrate each user needs (requirements) and a design for each need (requirement)
 
 ## Input Value
-  - Users needs to identify how to input a duration value. For example, if a user needs to format `1000 seconds`, how could the user pass the value to the formatting function.
+
+  + Users needs to identify how to input a duration value. For example, if a user needs to format `1000 seconds` , how could the user pass the value to the formatting function.
```
Comment (ryzokuken) [inline README.md:42]:
```suggestion
  + Users need to identify how to input a duration value. For example, if a user needs to format `1000 seconds` , how could the user pass the value to the formatting function.
```

File: README.md [hunk 6]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
+| Dotted       | 1: 50: 00             |
 
 ### Design
-  * The user can determine the formatting width using a  parameter `style` and the value of this parameter take a ` string` as following:
-      * `“wide”`
-      * `“short”`
-      * `“narrow”`
-      * `“dotted”`
+
+  + The user can determine the formatting width using a  parameter `style` and the value of this parameter take a ` string` as following:
```
Comment (ryzokuken) [inline README.md:65]:
```suggestion
  + The user can determine the formatting width using a parameter `style` and the value of this parameter may be one of the following `string`s:
```

File: README.md [hunk 7]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
-  * Second
-  * Millisecond
-  * Microsecond
-  * Nano Second
+
+* We are going to support the same fields as in [ `Temporal.Duration` ](https://github.com/tc39/proposal-temporal/blob/main/docs/duration.md), which contains:
+
+   * years
+   * months
+   * days
```
Comment (ryzokuken) [inline README.md:92]: weeks

File: README.md [hunk 8]
```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
...
-    * `"trailingOnly"` 
-    * `"none"` // Do not hide any zero-valued fields.
 
-- Default Value: `"none"`
+* Users can set the `hideZeroValues` parameter  by one of the following values:
+    - `"all"` // Hide all the fields that have a zero-valued.
+    - `"leadingAndTrailing"` // Hide all the zero fields in the leading or the training
+    - `"leadingOnly"` 
+    - `"trailingOnly"` 
+    - `"none"` // Do not hide any zero-valued fields.
```
Comment (ryzokuken) [inline README.md:151]: These don't need to be comments.

File: README.md (hunk 1 above)
Comment (younies) [reply README.md:16]: I used a markdown plugin from VSCode and you can clearly see the bullets in the preview mood.

File: README.md (hunk 2 above)
Comment (younies) [reply README.md:21]: ditto.

File: README.md (hunk 4 above)
Comment (younies) [reply README.md:34]: ditto.

File: README.md (hunk 5 above)
Comment (younies) [reply README.md]: Done!

File: README.md (hunk 6 above)
Comment (younies) [reply README.md]: Done!

File: README.md (hunk 7 above)
Comment (younies) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (ryzokuken) [reply README.md:16]: I misunderstood.

**Pull request merged**